docs/dxp/7.x/en/site-building/creating-pages/building-and-managing-content-pages/fragment-styles-reference.md

```diff
@@ -0,0 +1,107 @@
+| Padding Right | The _Padding Right_ property sets the right padding of an element. |
+
+## Size
+
+Configure the element's dimensions.
+
+| Field | Description |
+| --- | --- |
+| Width | This is the width of the element. |
+| Height | This is the height of the element. |
+| Min Width | If scaling, this is the minimum width. |
+| Max Width | If scaling, this is the maximum width. |
+| Min Height | If scaling, this is the minimum height. |
+| Max Height | If scaling, this is the maximum height. |
+
+### Overflow
+
+The overflow property specifies what should happen if content overflows an element's box.
+
+This property specifies whether to clip content or to add scrollbars when an element's content is too big to fit in a specified area.
+
+| Field | Description |
+| --- | --- |
+| Default | The default is _Visible_ (see below). |
+| Visible | The overflow is not clipped. It renders outside the element's box. |
+| Hidden | The overflow is clipped, and the rest of the content will be invisible. |
+| Scroll | The overflow is clipped, but a scroll-bar is added to see the rest of the content. |
+| Auto | If overflow is clipped, a scroll-bar should be added to see the rest of the content. |
+
+## Text
+
+![You can configure how the element texts appear.](./fragment-styles-reference/images/02.png)
+
+| Field | Description |
+| --- | --- |
+| Font Family | The _Font Family_ property specifies the font for an element. |
+| Font Weight | The _Font Weight_ property sets how thick or thin characters in text should be displayed. |
+| Font Size | The _Font Size_ property sets the size of a font. |
+| Text Color | Choose a color for the text. |
+| Text Align | The _Text Align_ property specifies the horizontal alignment of text in an element. Choose from _Left_, _Right_, _Center_, or _Justify_. |
+
+## Background
+
+| Field | Description |
+| --- | --- |
+| Background Color | Choose a background color from the palette. |
+| Image Source | If set to _Manual Selection_, users can upload a background image. If set to _Content Mapping_, users can choose from a Web Content, Document and Media, or a Blog asset. |
+
+## Borders
+
+| Field | Description |
+| --- | --- |
+| Border Width | The _Border Width_ property sets the width of an element's four borders. |
+| Border Radius | The _Border Radius_ property defines the radius of the element's corners and allows you to add rounded corners. |
+| Border Color | The _Border Color_ property sets the color of an element's four borders. |
+
+## Effects
+
+| Field | Description |
+| --- | --- |
+| Opacity | This sets the transparency level; 100 is default and 0 is transparent. |
+| Shadow | This adds a shadow around the box; can choose from a small, medium, or large shadow. |
+
+## Additional Information
+
+* [Configuring Fragment Styles](./configuring-fragment-styles.md)
+* [Content Pages Overview](./content-pages-overview.md)
+* [Using Fragments](./using-fragments.md)
```

Comment:
This is also super well explained! 🥳 The only thing I miss is the relationship that exists between some of the styles and the Style Book tokens. I'm copy-pasting from the doc draft I wrote some weeks ago:

_Something important is that some of the common styles are linked to Style Book tokens. Por example, the styles of type color show in the palette all the colors present in the Style Book, and other styles (specifically font family, font size, font weight, border radius and shadow) are also linked to tokens. It means that, for example, setting a fragment font size to “Font Size Sm”, give it the value that the token fontSizeSm has in the Style Book.
It could be possible that a site have set a theme that doesn’t have a token definition.  It won’t fail or throw any error, as we have taken this case into account, but for those cases, some of the common styles may not work as expected._

The common styles definition is [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-api/src/main/resources/com/liferay/layout/util/structure/common-styles.json). There you can see how some of the style values have a `frontendTokenName` property, that indicates which specific token it is linked to.

Reply:
Hi Sandro, we don't have anything for Style Book yet. Adding this to our to-do.